3/Destructuring.js

```diff
@@ -0,0 +1,19 @@
+const sarah = {
+  name: 'Sarah',
+  job: {
+    name: 'Andersen',
+    hiringDate: '23.07.2020'
+  }
+}
+
+function printJobName(user) {
+  const {
```

Comment:
creating additional variables not needed. Manipulate this changes with input params please